examples/src/multi-file/endpoints/get-user.ts

```diff
@@ -7,7 +7,7 @@ export class Api {
     path: "/users/:userId"
   })
   getUser(
-    @pathParam userId: string
+    @pathParam() userId: string
```

Comment:
should we make the example `multi-file` and `single-file` to be consistent? this one does not have description, while the one in the single file has description